src/engines/html5/media-source/adapters/fairplay-drm-handler.js

```diff
@@ -34,7 +34,9 @@ class FairplayDrmHandler {
       responseFilter: (type, response) => {
         let responseObj = {};
         try {
-          let keyText = response.data.trim();
```

Comment:
should keep the trim in case server returns trailing spaces - or check with uDRM team if this is not necessary anymore

Reply:
Right, better to keep it anyway it'll safe